api/event.ts

```diff
@@ -0,0 +1,21 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node'
+import { Group, MeetupEvent } from 'types'
+const groupData = require('../data/data.json')
+
+const GROUP_DATA: Array<any> = (groupData as any).data.groupByUrlname.pastEvents
+  .edges
+
+const _getEvent = (id: string): MeetupEvent => {
+  console.log(id)
+
+  let node = GROUP_DATA.find((event) => event.node.id === id)
```

Comment:
you can return directly the node, no need for a new var

Reply:
cool!